src/js/_carousel.js

```diff
@@ -0,0 +1,41 @@
+const indicators = [...document.querySelectorAll('.card-indicators button')]
+const card = [...document.querySelectorAll('.card')]
+const btnFor = document.querySelector('.control__arrow-for')
+const btnBack = document.querySelector('.control__arrow-back')
+
+let actualStory = 0 // Default 0
+
+indicators[0].classList.add('active')
+card[0].classList.add('active')
+
+btnFor.addEventListener('click', () => {
+  indicators[actualStory].classList.remove('active')
```

Comment:
Instead of remove and add you can use toggle. 
https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle

Reply:
Very nice advice, a lot of thanks.